samples/vboxwrapper/vbox_vboxmanage.cpp

```diff
@@ -353,7 +353,12 @@ int VBOX_VM::create_vm() {
     //
     vboxlog_msg("Disabling Audio Support for VM.");
     command  = "modifyvm \"" + vm_name + "\" ";
-    command += "--audio none ";
+    if (is_virtualbox_version_newer(7, 0, 4)) {
+        command += "--audio-enabled off";
```

Comment:
Later some other arguments are added to the `command` value, please add a space at the end of the string